packages/engine-server/src/drivers/sqlite/tables/HierarchyTableUtils.ts

```diff
@@ -5,7 +5,7 @@ import { SqliteError } from "../SqliteError";
 import { SqliteQueryUtils } from "../SqliteQueryUtils";
 
 export class HierarchyTableRow {
-  constructor(public parentId: string, public childId: string) {}
```

Comment:
I had actually renamed from parent/child -> parentId/childId in my first PR in response to a code review comment.  Was it intentional to drop the 'id' part or just a merge artifact?

Reply:
I changed it because all the queries seem to insert them as `parent` and `child`. I just assumed the tables were defined correctly.

Reply:
I'll change it back to `parentId` and `childId` and instead change the queries so the column names follow.